cpp/matrix.hpp

```diff
@@ -199,4 +204,102 @@ template<class T> struct Matrix {
         ret.assign(v[i].begin(), v[i].end());
         return ret;
     }
+    /**
+     * @brief 保持している行列に行列Bを足す
+     * @param B 足す行列
+     * @return @c *this
+    */
+    constexpr Matrix &operator+=(const Matrix &B) {
+        if(n == 0) return (*this);
+        assert(n == B.size() && m == B[0].length());
```

Comment:
```suggestion
        assert(n == B.size() && m == B[0].size());
```
std::vectorに`length()`はない